lib/withAuth.ts

```diff
@@ -32,6 +31,8 @@ export async function getUserFromToken(
 
   const decoded = jwt.verify(token, process.env.JWT_SECRET) as JwtPayload;
   const userId = decoded.userId as string;
+  // dynamically import newrelic
+  const newrelic = await import('newrelic');
```

Comment:
this helps resolve some of the Max emitter warnings / memory leak from running tests